CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.0]
+
+### `Added`
+
+- `--provider` parameter to be used to pick where to fetch data from and with what tool
+  - `--provider SRA` runs the current pipeline path
+  - `--provider ENA` runs fastq-dl to fetch data
```

Comment:
For clarity, I think it might be better to say what each pulls from, even if it's obvious for some:

- `--provider SRA` fetches data from the Sequence Read Archive
- `--provider ENA` fetches data from the European Nucleotide Archive